src/dashboard/frontend/package.json

```diff
@@ -51,8 +51,16 @@
     "base64-helpers": "git+https://github.com/artefactual-labs/base64-helpers.git",
     "d3": "^3.5.12",
     "font-awesome": "^4.3.0",
-    "lodash": "^4.5.1",
+    "lodash": "^4.17.12",
     "moment": "^2.21.0",
     "restangular": "^1.5.1"
-  }
+  },
+   "overrides": {
+    "fsevents": "1.2.13",
+    "js-yaml": "3.13.1",
+    "json5": "1.0.2",
+    "webpack": {
+      "loader-utils": "1.4.2"
+    }
+    }
```

Comment:
Indentation seems a bit off?

Reply:
I wonder if we should add this file to the pre-commit hook that formats other files in the repo? https://github.com/artefactual/archivematica/blob/155f229fa7a15cc7f01a57eb5a9fc0335a20f5c8/.pre-commit-config.yaml#L7-L11

Reply:
I tested this and the hook works as expected but I'm hesitant about its default behavior of sorting mapping keys. It also has a couple of [arguments](https://github.com/pre-commit/pre-commit-hooks#pretty-format-json) that could be useful: `--no-sort-keys` to get the file formatted with no key reordering and `--top-keys` which allows you to define keys you want to keep in place. We could use the latter for the metadata keys of the package such as `name`, `description` and `version`. In any case we can discuss it after this PR gets merged :slightly_smiling_face: